dartagnan/src/main/java/com/dat3m/dartagnan/encoding/WmmEncoder.java

```diff
@@ -649,9 +649,8 @@ public Void visitSyncBarrier(SyncBar syncBar) {
         @Override
         public Void visitSyncFence(SyncFence syncFenceDef) {
             final Relation syncFence = syncFenceDef.getDefinedRelation();
-            ;
             final boolean idl = !context.useSATEncoding;
-            final String relName = syncFence.getName().get(); // syncFence is base, it always has a name
+            final String relName = syncFence.getName().orElseThrow(); // syncFence is base, it always has a name
```

Comment:
Any special reason for the change? the comment explains that `get()` is always guaranteed to succeed.

Reply:
The reason is to suppress the warning by the IDE and marginally improve readability. `orElseThrow()` is equivalent to `get()` but makes the intent more explicit

Reply:
See [this](https://bugs.openjdk.org/browse/JDK-8140281)